extensions/shared/upgrade-nudge/style.scss

```diff
@@ -0,0 +1,85 @@
+@media ( max-width: 600px ) {
+	// stylelint-disable-next-line selector-max-id
+	.is-iframed #wpbody {
+		padding-top: 0;
+	}
+}
+
+@media ( min-width: 600px ) {
+	body.is-iframed.is-fullscreen-mode .edit-post-header {
+		top: 0;
+	}
+}
+
+@media ( min-width: 782px ) {
+	.edit-post-layout__content {
+		position: fixed;
+		bottom: 0;
+		left: 0;
+		right: 0;
+		top: 88px;
+		min-height: calc( 100% - 88px );
+		height: auto;
+		margin-left: 160px;
+	}
+
+	body.is-fullscreen-mode .edit-post-layout__content {
+		top: 56px;
+		position: fixed;
+	}
+}
+
+@media ( min-width: 782px ) and ( max-width: 1079px ) {
+	.is-fullscreen-mode .edit-post-layout.has-fixed-toolbar .components-notice {
+		top: 37px;
+	}
+}
+
+.components-notice-list {
+	z-index: 29; // Ensure notices are placed behind the editor header (z-index: 30).
+}
+
+.paid-block__wrapper {
+	border: 1px solid var( --color-border-subtle );
+}
+
+.paid-block__disabled {
+	opacity: 0.5;
+	padding: 10px;
+}
+
+.upgrade-nudge {
+	border-left: 3px solid var( --color-plan-premium );
```

Comment:
Eventually we'll need variations of nudges for different plan levels, right? `.upgrade-nudge.is-premium`, etc.

Reply:
Turns out that Calypso's `UpgradeNudge` component's styling [only uses the Premium Plan's colors](https://github.com/Automattic/wp-calypso/blob/80bd51ee13d9c41e4e23e47c9e1fccc6a1abb393/client/blocks/upgrade-nudge/style.scss#L2), too -- regardless of what plan we're actually nudging the user to upgrade to. From a design perspective, one could probably argue with visual consistency, and general connotation of that 'gold' color?

So I guess I'm not making any provisions for different plan colors (selectors) here, at least as long as Calypso doesn't either.